openghg/store/_boundary_conditions.py

```diff
@@ -258,11 +255,12 @@ def read_file(
             #     metastore=metastore,
             #     update_keys=update_keys,
             # )
-
+            results["processed"][filepath.name] = datasource_uuids
```

Comment:
Is there another key besides "processed"? (Either for boundary conditions, or for other data classes?)